roles/lib_openshift/src/doc/ca_server_cert

```diff
@@ -0,0 +1,90 @@
+# flake8: noqa
+# pylint: skip-file
+
+DOCUMENTATION = '''
+---
+module: oc_adm_ca_server_cert
+short_description: Module to run openshift oc adm ca create-server-cert
+description:
+  - Wrapper around the openshift `oc adm ca create-server-cert` command.
+options:
+  state:
+    description:
+    - Present is the only supported state.  The state present means that `oc adm ca` will generate a certificate
+    - and verify if the hostnames and the ClusterIP exists in the certificate.
+    - When create-server-cert is desired then the following parameters are passed.
+    - ['cert', 'key', 'signer_cert', 'signer_key', 'signer_serial']
+    required: false
+    default: present
```

Comment:
@detiber, I'd image a delete would be a `file: state=absent`.  I'm just trying to duplicate the `oc adm ca create-server-cert` command.

Good suggestion on backups.  We could definitely add that support.